include/translatable-objects.php

```diff
@@ -98,13 +98,14 @@ public function get_secondary_translatable_objects() {
 	 * @param string[] $filter An array on value to filter taxonomy names to return.
 	 * @return string[] Taxonomy names.
 	 *
-	 * @phpstan-param array<'language'|'translations'> $filter
+	 * @phpstan-param array<'language'|'translations'|'secondary'> $filter
 	 * @phpstan-return list<non-empty-string>
 	 */
 	public function get_taxonomy_names( $filter = array( 'language', 'translations' ) ) {
 		$taxonomies = array();
+		$objects    = in_array( 'secondary', $filter, true ) ? $this->get_secondary_translatable_objects() : $this->objects;
 
-		foreach ( $this->objects as $object ) {
+		foreach ( $objects as $object ) {
```

Comment:
Not sure we should keep this change. Apart that you don't use it anymore, the `secondary` filter is not used as the othesr.
- `array( 'language', 'translations' )` returns the union.
- `array( 'language', 'secondary' )` returns the intersection.

Reply:
Reverted in [0f1fa8e](https://github.com/polylang/polylang/pull/1176/commits/0f1fa8e99b47bc3a80609878fc32b054988b7728).